clog/scribe_log.avsc

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "record",
+    "namespace": "scribe_log",
+    "name": "<replace_with_scribe_log_name>",
+    "doc": "Schema used to serialize scribe messages. This schema won't be actually used for scribe logs, and no stream will be assigned to this schema.",
```

Comment:
I would rephrase this as: "This schema is not used to actually serialize messages."